### PR TITLE
Removing  unneeded intent tests

### PR DESCRIPTION
**Pull request by TurtleSquad007**

### CODE REVIEW

**Comment by giancarlopernudisegura** [comment]
This seems to have some of the fixes I'm working on in #200

**Comment by TurtleSquad007** [comment]
Should we merge?

**Comment by giancarlopernudisegura** [comment]
We should merge the tests involving experiments to the other branch.

**Comment by TurtleSquad007** [comment]
We'll have to meet up and decide. I just tried to merge but we had too many conflicts. Let's meet up at some point to day and deal with them.

**Comment by TurtleSquad007** [comment]
Not getting anymore conflicts

File: code/app/src/main/java/com/example/experiment_automata/backend/experiments/Experiment.java [hunk 1]
```diff
@@ -68,6 +68,7 @@ protected Experiment(String description, int minTrials, boolean requireLocation,
         this.type = type;
         this.enableFirestore = enableFirestore;
         results = new ArrayList<>();
+        testMode = false;
```
Comment (Pegmode) [inline code/app/src/main/java/com/example/experiment_automata/backend/experiments/Experiment.java:71]: Issue with this line. testMode not  in this scope. While both testing and running the app

`\Experiment-Automata\code\app\src\main\java\com\example\experiment_automata\backend\experiments\Experiment.java:71: error: cannot find symbol
        testMode = false;
        ^
  symbol:   variable testMode
  location: class Experiment<T>
  where T is a type-variable:
    T extends Trial<?> declared in class Experiment`

**Review by Pegmode — CHANGES_REQUESTED** [review]

File: code/app/src/main/java/com/example/experiment_automata/backend/experiments/Experiment.java (hunk 1 above)
Comment (TurtleSquad007) [reply code/app/src/main/java/com/example/experiment_automata/backend/experiments/Experiment.java:71]: Whoops something went wrong while merging let me take care of it. I'll put this as a draft in the mean time

**Review by Pegmode — APPROVED** [review]
Builds/Runs fine now.

**Pull request closed without merging**